manimlib/mobject/types/vectorized_mobject.py

```diff
@@ -595,35 +595,69 @@ def consider_points_equals(self, p0, p1):
             atol=self.tolerance_for_point_equality
         )
 
+    def consider_points_equals_2d(self, p0, p1):
+        """
+        Determine if two points are close enough to be considered equal.
+
+        This uses the algorithm from np.isclose(), but expanded here for the
+        2D point case. NumPy is overkill for such a small question.
+        """
+        rtol = 1.e-5  # default from np.isclose()
```

Comment:
This should use `self.tolerance_for_point_equality`

Reply:
`np.isclose()` takes two optional tolerances: atol and rtol. The original `consider_points_equals()` passes `self.tolerance_for_point_equality` for atol, and leaves rtol it's default value. I have done the same here; see lines 605 and 606 together.

`np.isclose()` def: https://github.com/numpy/numpy/blob/master/numpy/core/numeric.py#L2167